tests/pytests/test_debug_commands.py

```diff
@@ -359,3 +361,15 @@ def testSpecIndexesInfo(env: Env):
     expected_reply["inverted_indexes_memory"] = getInvertedIndexInitialSize(env, ['NUMERIC'])
     debug_output = env.cmd(debug_cmd(), 'SPEC_INVIDXES_INFO', 'idx')
     env.assertEqual(to_dict(debug_output), expected_reply)
+
+def testIndexes(env: Env):
+    env.expect('FT.CREATE', 'idx', 'SCHEMA', 'name', 'TEXT').ok()
+    debug_output = env.cmd(debug_cmd(), 'INDEXES')
+    env.assertEqual(debug_output, ['Index@4e7f626df794f6491574a236f22c100c34ed804f'])
+
+def testIndexObfuscatedInfo(env: Env):
```

Comment:
Test also the fields of the info response? At least the interesting ones relative to this PR, like the fields of the index.

Reply:
Prefixes as well